### PR TITLE
Allow to install latest version, closes #1

### LINKED ISSUE #1: Check docker_compose_version to latest from internet if not provided
This should be the default behavior, instead of an old fixed version

### PR DESCRIPTION
**Pull request by bkmeneguello**

### CODE REVIEW

**Comment by Bessonov** [comment]
LGTM, but two suggestions:

1. It's better to fail if no version is set *or*, like you proposed, to use 'latest' by default. I like the first, but I think your proposal is OK forme
2. Maybe it's useful to add info about latest in README.

**Comment by bkmeneguello** [comment]
I agree with failing if the version is absent, but in this case, the default value should be removed, if you agree I'll update the PR

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Requires docker. See also [ansible docker role](https://galaxy.ansible.com/Besso
 Role Variables
 --------------
 
-(optional) `docker_compose_version` specifies docker-compose version.
+(required) `docker_compose_version` specifies docker-compose version. Allow the keyworkd `latest` which queries docker-compose Github repo for latest release version.
```
Comment (Bessonov) [inline README.md]: small typo `keyworkd`

**Comment by Bessonov** [comment]
It's up to you, I'm fine with both.

File: README.md (hunk 1 above)
Comment (bkmeneguello) [reply README.md]: fixed

**Pull request merged**